compiler/circle-mpqsolver/README.md

```diff
@@ -0,0 +1,16 @@
+# circle-mpqsolver
+
+_circle-mpqsolver_ provides light-weight methods for finding optimal Mixed Precision Quantization model, such that it is as accurate as _float_ model and as fast as _int8_ model.
```

Comment:
please split the line so that editing in 80 or 100 columns is possible

Reply:
> finding optimal

is this true or is it `sub optimal` ?

Reply:
currently looks like
![image](https://user-images.githubusercontent.com/4616940/207721098-3b100fb0-c221-489a-912b-63213f38bb6c.png)

Reply:
Better to remove the word `optimal`. Also, the below sentence seems like an overstatement.

```
such that it is as accurate as _float_ model and as fast as _int8_ model.
```

I slightly rephrased the sentence.

_circle-mpqsolver_ provides light-weight methods for finding a high-quality mixed-precision model within a reasonable time.